src/screens/app/operations/charges-operations.screen.tsx

```diff
@@ -0,0 +1,65 @@
+import { useMemo, useState } from "react"
+
+import { Input } from "@/components/ui/input"
+import { Card, CardContent } from "@/components/ui/card"
+import { ContainedScreen } from "@/components/prefab/contained-screen"
+import { PaginationButtons, usePagination } from "@/components/shared/pagination"
+
+import { useChargesOperations } from "@/features/token-operations/hooks"
+import { useParams } from "react-router-dom"
+import { useEmptyStateContent, useLoadingStateContent } from "./hooks"
+import { ChargeOperationTable } from "@/features/token-operations/components"
+
+export const ChargesOperationsScreen = () => {
+  const { id } = useParams()
+  const {data: chargesOperations = [], isFetching, isFetched } = useChargesOperations(id)
+
+  const [filterValue, setFilterValue] = useState("")
+
+  const operationsFilteredByValue = useMemo(() => {
```

Comment:
Since we won't have the all the rows locally anymore, the filters should be sent to the API